### ISSUE TITLE
Warning in Release build on Zeppelin

### ISSUE THREAD

**Issue by nsoblath** [project8/psyllid#29]
Recorded here for posterity and fixing at some point.

```
Scanning dependencies of target PsyllidDAQ
[ 86%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/event_builder.cc.o
[ 87%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/frequency_mask_trigger.cc.o
[ 88%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/packet_receiver_socket.cc.o
/home/project8/Repos/psyllid/source/daq/event_builder.cc: In member function ‘virtual void psyllid::event_builder::execute(midge::diptera*)’:
/home/project8/Repos/psyllid/source/daq/event_builder.cc:54:22: warning: variable ‘t_current_id’ set but not used [-Wunused-but-set-variable]
             uint64_t t_current_id = 0;
                      ^
[ 88%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/roach_config.cc.o
[ 89%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/streaming_writer.cc.o
[ 90%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/terminator.cc.o
[ 90%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/tf_roach_monitor.cc.o
[ 91%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/tf_roach_receiver.cc.o
[ 91%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/triggered_writer.cc.o
[ 92%] Building CXX object source/daq/CMakeFiles/PsyllidDAQ.dir/packet_receiver_fpa.cc.o
/home/project8/Repos/psyllid/source/daq/packet_receiver_fpa.cc: In member function ‘virtual void psyllid::packet_receiver_fpa::initialize()’:
/home/project8/Repos/psyllid/source/daq/packet_receiver_fpa.cc:109:14: warning: variable ‘test’ set but not used [-Wunused-but-set-variable]
         bool test = f_ring.f_rd == nullptr;
              ^
/home/project8/Repos/psyllid/source/daq/triggered_writer.cc: In member function ‘void psyllid::triggered_writer::exe_loop_is_running(psyllid::triggered_writer::exe_loop_context&)’:
/home/project8/Repos/psyllid/source/daq/triggered_writer.cc:357:75: warning: suggest braces around empty body in an ‘if’ statement [-Wempty-body]
                     if( a_ctx.f_is_new_event ) LDEBUG( plog, "New event" );
                                                                           ^
```

**Comment by nsoblath** [comment]
Another warning:
```
/home/project8/Repos/psyllid/source/daq/triggered_writer.cc: In member function 'void psyllid::triggered_writer::exe_loop_is_running(psyllid
::triggered_writer::exe_loop_context&)':
/home/project8/Repos/psyllid/source/daq/triggered_writer.cc:357:75: warning: suggest braces around empty body in an 'if' statement [-Wempty-
body]
                     if( a_ctx.f_is_new_event ) LDEBUG( plog, "New event" );
```

**Comment by nsoblath** [comment]
Solved in 7b1007b2accb33c0ae294ef923260f439454f462
